weave-js/src/components/PagePanel.tsx

```diff
@@ -110,6 +112,8 @@ const JupyterControlsIcon = styled.div`
   }
 `;
 
+const HOST_SESSION_ID_COOKIE = `host_session_id`;
```

Comment:
should this be namespaced with wandb / weave?

Reply:
I don't think we need to do that, this is the same cookie that the App uses.